stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/GeneralApi.kt

```diff
@@ -0,0 +1,13 @@
+package io.getstream.chat.android.client.api2
+
+import io.getstream.chat.android.client.api.AuthenticatedApi
+import io.getstream.chat.android.client.call.RetrofitCall
+import okhttp3.ResponseBody
+import retrofit2.http.OPTIONS
+
+@MoshiApi
+@AuthenticatedApi
+internal interface GeneralApi {
```

Comment:
Didn't find a better name for these miscellaneous API calls, things like `sync` will also come here later